fact_plots/bias_resolution.py

```diff
@@ -74,9 +76,9 @@ def plot_bias_resolution(
         binned['bias'],
         xerr=0.5 * binned['width'],
         yerr=binned['bias_err'],
-        label='Bias',
+        label=kwarg_or_default(kwargs, 'bias_label', 'Bias'),
         linestyle='',
-        color='C0'
+        color=kwarg_or_default(kwargs, 'bias_color', 'C0'),
```

Comment:
This could be set using a matplotlibrc, my original idea was to add as few code as possible for things that are configurable via matplotlibrc, so to change this color, use a matplotlibrc where `C0` is another color by setting `axes.prop_cycle`

Reply:
done